spring-cloud-sleuth-zipkin/src/main/java/org/springframework/cloud/sleuth/zipkin2/ZipkinProperties.java

```diff
@@ -37,6 +37,13 @@
 	 */
 	private String baseUrl = "http://localhost:9411/";
 
+	/**
+	 * Api path that will be appended to baseUrl above as suffix This is applicable if you
+	 * are using other monitoring tools. IE, new relic, the trace api doesn't need api
```

Comment:
Change to:

use other monitoring tools, such as New Relic. The trace API doesn't need the API